### ISSUE TITLE
Creating an `OnnxTensor` requires a `uint8` type.

### ISSUE THREAD

**Issue by Myhuiku** [Ahwar/ONNX-NER-with-Java#2]
Hello, my NER model has two inputs and one output:

input_ids

Name: input_ids
Tensor: int64[batch_size, max_seq_len]
attention_mask

Name: attention_mask
Tensor: uint8[batch_size, max_seq_len]
logits

Name: logits
Tensor: int64[batch_size, max_seq_len]
The max_seq_len is 512. 
How can I create a UINT8 tensor for the attention mask and who to define the shape?
